README.md

```diff
@@ -194,6 +194,176 @@ To use the Flow Token contract as is, you need to follow these steps:
+            (from: FungibleTokenSwitchboard.StoragePath) 
+              ?? panic("Could not borrow reference to switchboard")
+        }
+
+        execute {
+          // Add the capability to the switchboard using addNewVault method
+          self.switchboardRef.addNewVault(capability: self.exampleTokenVaultCapabilty)
+        }
+    }
+    ```
+    This function will panic if is not possible to `.borrow()` a reference to a `&{FungibleToken.Receiver}` from the passed capability. It will also panic if there is already a capability stored for the same `Type` of resource exposed by the capability.
+
+ 1. Adding one or more capabilities using the paths where they are stored using `addNewVaultsByPath(paths: [PublicPath], address: Address)`
+    * When using this method, an array of `PublicPath` objects should be pass along with the `Address` of the account from where the vaults' capabilities should be retrieved.
+
+    ```cadence
+    transaction (address: Address) {
+
+        let exampleTokenVaultPath: PublicPath
+        let vaultPaths: [PublicPath]
+        let switchboardRef:  &FungibleTokenSwitchboard.Switchboard
+
+        prepare(signer: AuthAccount) {
+          // Get the example token vault path from the contract
+          self.exampleTokenVaultPath = ExampleToken.ReceiverPublicPath
+          // And store it in the array of public paths that will be passed to the
+          // switchboard method
+          self.vaultPaths = []
+          self.vaultPaths.append(self.exampleTokenVaultPath)
+          // Get a reference to the signers switchboard
+          self.switchboardRef = signer.borrow<&FungibleTokenSwitchboard.Switchboard>
+            (from: FungibleTokenSwitchboard.StoragePath) 
+              ?? panic("Could not borrow reference to switchboard")
+        }
+
+        execute {
+          // Add the capability to the switchboard using addNewVault method
+          self.switchboardRef.addNewVaultsByPath(paths: self.vaultPaths, 
+                                                        address: address)
+        }
+    }
+    ```
+    This function won't panic, instead it will just not add to the `@Switchboard` any capability which can not be retrieved from any of the provided `PublicPath`s. It will also ignore any type of `&{FungibleToken.Receiver}` that is already present on the `@Switchboard`
+
+ ## Removing a vault from the switchboard
+ If a user no longer wants to be able to receive deposits from a certain FT, or if they want to update the provided capability for one of them, they will need to remove the vault from the switchboard. This can me acomplish by using `removeVault(capability: Capability<&{FungibleToken.Receiver}>)`. As for adding new vaults it is required to first retrieve the vault's capability from the user account and passing it as a parameter. This can be observed in the template transaction `transactions/switchboard/remove_vault_capability.cdc`:
```

Comment:
```suggestion
 If a user no longer wants to be able to receive deposits from a certain FT, or if they want to update the provided capability for one of them, they will need to remove the vault from the switchboard.
 This can be accomplished by using `removeVault(capability: Capability<&{FungibleToken.Receiver}>)`. 
This can be observed in the template transaction `transactions/switchboard/remove_vault_capability.cdc`:
```